### ISSUE TITLE
Run error

### ISSUE THREAD

**Issue by chaobian** [Ruturaj123/Flowchart-Detection#14]
object_detection/inference_graph/frozen_inference_graph.pb; No such file or directory

**Comment by mastermaddy** [comment]
Yes, I found the same issue, is there any fix to this? I changed the path as well but then some Parsing Error is coming.
